### DISCUSSION TITLE
Windows 10 Installation

### DISCUSSION THREAD

**Discussion by RichardScottOZ** [gempy-project/gempy#601] [General]
Just trying again - install libpython m2w64toolchain 
then theano 
then mkl-service 
then upgrade theano via pip

then came across this dll error https://github.com/conda/conda/issues/6396#issuecomment-350254762
https://github.com/conda/conda/issues/6396

Comment on #6396 scipy - ImportError: DLL load failed: The specified module could not be found.
<https://github.com/conda/conda|conda/conda>conda/conda | Dec 8th, 2017 | Added by GitHub

#6396 scipy - ImportError: DLL load failed: The specified module could not be found.
<https://github.com/conda/conda|conda/conda>conda/conda | Dec 5th, 2017 | Added by GitHub

then installed jupyter - which presumably snagged the conda scipy/numpy again - so had to redo the above force remove scipy/numpy from conda and install via pip again

e.g. each conda install of something relevent to them might make you do it again to get it to work

for reference, if trying this again in a year

**Comment by Leguark** [comment]
installing m2w64toolchain and mkl-service is only necessary for using `cuda`. I would not get into there if you do not have to. If you have to, first make sure that `theano` works on the gpu before installing `gempy` on top of it
